src/Prometheus.Client.HttpClient/Constants.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Prometheus.Client.HttpClient
+{
+    public static class Constants
```

Comment:
Mark as `Internal`, rename to `Defaults`.

Reply:
@phnx47 
Can't mark as internal since same constants are being used by test project as well. Thought about using `InternalsVisibileTo` but thought it'd be an overkill.

Reply:
Yes, mark as `InternalsVisibileTo`

Example: https://github.com/prom-client-net/prom-client-aspnetcore/blob/2176ab9628025664dad605bdd5f0c0e66c46ba39/src/Prometheus.Client.AspNetCore.csproj#L13